cmd/ocm-backplane/login/login.go

```diff
@@ -101,12 +117,12 @@ func runLogin(cmd *cobra.Command, argv []string) (err error) {
 	}
 
 	// Get Backplane URL
-	if args.backplaneURL == "" {
-		args.backplaneURL, err = utils.DefaultOCMInterface.GetBackplaneURL()
-		if err != nil || args.backplaneURL == "" {
+	if globalOpts.BackplaneURL == "" {
+		globalOpts.BackplaneURL, err = utils.DefaultOCMInterface.GetBackplaneURL()
```

Comment:
We should move this `GetBackplaneURL` func out of OCM interface? it's not part of OCM.

Reply:
Done